electric-motor-torque-curve/index.html

```diff
@@ -10,8 +10,7 @@ <h1>Input</h1>
 
 <p>
 
-
-  <label for ="calc-1">First Calc:</label>
+  <label for ="calc-1">Voltage (Rated):</label>
```

Comment:
This label is labeled with ID (for) attribute "calc-1" but the others match the name of the contents within the tag like "no_load_speed". Please fix all of the label names so they match their corresponding input fields.